pkg/kwokctl/pki/pki.go

```diff
@@ -17,36 +17,61 @@ limitations under the License.
 package pki
 
 import (
-	_ "embed"
-	"os"
-
-	"sigs.k8s.io/kwok/pkg/kwokctl/utils"
+	"crypto"
+	"crypto/x509"
+	"net"
+	"time"
 )
 
-var (
-	//go:embed testdata/ca.crt
-	CACrt []byte
-	//go:embed testdata/admin.key
-	AdminKey []byte
-	//go:embed testdata/admin.crt
-	AdminCrt []byte
-)
+func generatePki() (caCert *x509.Certificate, caKey crypto.Signer, cert *x509.Certificate, key crypto.Signer, err error) {
```

Comment:
nit: can we create a struct capturing the pki info?

```go
type pki struct {
    cert *x509.Certificate
    key crypto.Signer
    caCert *x509.Certificate
    caKey crypto.Signer
}
```
it will be easier to pass this around and add/remove members if needed.